sdk/rust/oak/src/logger/mod.rs

```diff
@@ -53,17 +35,15 @@ impl Log for OakChannelLogger {
         if !self.enabled(record.metadata()) {
             return;
         }
-        let log_entry = LogEntry {
-            // We add a newline to the message to force flushing when printed by the host.
-            message: format!(
-                "{}  {} : {} : {}\n",
-                record.level(),
-                record.file().unwrap_or_default(),
-                record.line().unwrap_or_default(),
-                record.args()
-            ),
+        let log_msg = crate::proto::logger::LogMessage {
+            file: record.file().unwrap_or_default().to_string(),
+            line: record.line().unwrap_or_default(),
+            level: map_level(record.level()),
+            message: format!("{}", record.args()),
+            cached_size: ::std::default::Default::default(),
+            unknown_fields: ::std::default::Default::default(),
```

Comment:
You should use the `..Default::default()` idiom to ensure that all fields are filled in, see elsewhere.

https://github.com/project-oak/oak/blob/4ad9891707b514703db37c45bbe532d94689af4c/examples/hello_world/module/rust/build.rs#L17-L24

Reply:
Nice idiom. Thanks :)